graylog2-web-interface/src/views/components/dashboard/DashboardsOverview/DashboardActions.tsx

```diff
@@ -52,6 +53,11 @@ const DashboardActions = ({ dashboard, refetchDashboards }: Props) => {
 
   return (
     <>
+      <InvestigationsMenu id={dashboard.id}
+                          type="dashboards"
+                          bsSize="xsmall"
+                          title={<span><Icon name="puzzle-piece" size="xs" /> Investigations</span>}
+                          pullRight />
```

Comment:
With this component we have an overflow problem in the dashboards overview:
![image](https://user-images.githubusercontent.com/46300478/224712264-7fb03470-d69e-4e2e-889a-f864975ed97c.png)

I had a similar problem in the past: https://github.com/Graylog2/graylog2-server/pull/14300 and fixed it by creating a `OverlayDropdown` component. 

Alternatively we could also create a `More Actions` dropdown and move `Delete` and "Add to investigation" into it, like we have it on other pages like the streams overview.

Reply:
OK. I'll do that, Add a `More Actions` menu with the delete and add to investigation options

Reply:
Looking into this issue. Looks like the root of the problem is the `EntityDataList` component. Setting `oveflow-x: 'auto'` for the wrapper element was also affecting `overflow-y`

Setting `overflow-x: 'clip'` prevents the table overflow the container div and it doesn't affect `overflow-y`, but there is no scroll option in the X axis.

I observed that the issue is at a very small break point, so we might need to create another PR to handle the overflow in the different break points.

One Idea could be to collapse all action under a dropdown using the `ellipsis` icon for the trigger. That would make the Actions column very small, preventing the overflow of the table.

Reply:
Unfortunately we can not rely on the breakpoint being small. The `EntityDataTable` is also being used on other pages and their content (the columns we display) is configurable. I think we need the horizontal scrolling.

My suggestion is to keep `overflow-x` and to use the `OverlayDropdown` button instead of the `DropdownButton`, since it displays the dropdown in a portal.